readme.md

```diff
@@ -22,6 +22,13 @@ git clone https://github.com/nkanaev/gofeed.git
 mv gofeed yarr
 cd yarr && make build_macos
 ```
+## run
+1) in terminal:
+```
+go run main.go
```

Comment:
wtf.... why run application code through a sub go runner? 
go run main.go
just compile it with `go build` or `make` and use the resulting `yarr` executable.

or do you think there is some advantage of using more memory by using the developer runner?